cmd/logger/logger.go

```diff
@@ -155,39 +184,24 @@ func getTrace(traceLevel int) []string {
 
 func logIf(level Level, err error, msg string,
 	data ...interface{}) {
-
-	isErrIgnored := func(err error) (ok bool) {
-		err = errors.Cause(err)
-		switch err.(type) {
-		case BucketNotFound, BucketNotEmpty, BucketExists:
-			ok = true
-		case ObjectNotFound, ObjectExistsAsDirectory:
-			ok = true
-		case BucketPolicyNotFound, InvalidUploadID:
-			ok = true
-		}
-		return ok
-	}
-
-	if err == nil || isErrIgnored(err) {
+	if err == nil {
 		return
 	}
 	// Get the cause for the Error
 	cause := err.Error()
 	// Get full stack trace
 	trace := getTrace(3)
 	// Get time
-	timeOfError := UTCNow().Format(time.RFC3339Nano)
+	timeOfError := time.Now().UTC().Format(time.RFC3339Nano)
 	// Output the formatted log message at console
 	var output string
 	message := fmt.Sprintf(msg, data...)
-	if log.json {
+	if jsonFlag {
 		logJSON, err := json.Marshal(&logEntry{
-			Level:   level.String(),
-			Message: message,
-			Time:    timeOfError,
-			Cause:   cause,
-			Trace:   trace,
+			Level: level.String(),
+			Time:  timeOfError,
+			Cause: cause,
+			Trace: traceEntry{Source: trace, Message: message},
 		})
 		if err != nil {
 			panic("json marshal of logEntry failed: " + err.Error())
```

Comment:
@harshavardhana's comment is also valid here:
`panic("json marshal of logEntry failed: " + err.Error())` => `panic(err)`

Reply:
This piece of code will be removed in the fatalIf rewrite.